alvr/server_openvr/src/lib.rs

```diff
@@ -202,14 +202,19 @@ extern "C" fn driver_ready_idle(set_default_chap: bool) {
                                 && ffi_right_hand_skeleton.is_some(),
                         };
 
-                        let body_motions = tracking::BODY_TRACKER_ID_MAP
-                            .keys()
-                            .filter_map(|id| {
-                                Some((*id, context.get_device_motion(*id, sample_timestamp)?))
-                            })
-                            .collect::<Vec<_>>();
-                        let ffi_body_trackers =
-                            tracking::to_ffi_body_trackers(&body_motions, track_body);
+                        let ffi_body_trackers = if track_body {
+                            tracking::BODY_TRACKER_IDS
+                                .iter()
+                                .filter_map(|id| {
```

Comment:
Why is this a filter_map?

Reply:
Because calling get_device_motion can fail with None. this happens when we pass a sample_timestamp which has not been recorded